packages/x-date-pickers/src/AdapterMomentJalaali/AdapterMomentJalaali.ts

```diff
@@ -146,6 +146,10 @@ export class AdapterMomentJalaali extends AdapterMoment implements MuiPickersAda
     return this.moment(value).locale('fa');
   };
 
+  public parseISO = (isoString: string) => {
```

Comment:
Was crashing, not sure why
This removed the `strict` attribute but since we are not using this method it's better than having it crash.